src/vstest.console/InProcessVsTestConsoleWrapper.cs

```diff
@@ -133,6 +136,30 @@ public InProcessVsTestConsoleWrapper(ConsoleParameters consoleParameters)
 
     internal ITestRequestManager? TestRequestManager { get; set; }
 
+    private void UpdateDotnetLookupPath()
+    {
+        const string dotnetRootOverrideEnvVarName = "VSTEST_WINAPPHOST_DOTNET_ROOT";
```

Comment:
This is not enough, you need to check for the x86 one too. See https://github.com/microsoft/vstest/blob/9a5bb2faf130809253b28d7ad2d434d785901025/src/Microsoft.TestPlatform.TestHostProvider/Hosting/DotnetTestHostManager.cs#L482-L489

Reply:
Removed this method altogether.